eservice/docs/basetypes.json

```diff
@@ -5,140 +5,101 @@
     "definitions": {
         "ecdsa-public-key": {
             "id": "#ecdsa-public-key",
-            "description": "an ECDSA public key from the SECP256k1 curve",
+            "description": [
+                "an ECDSA public key from the SECP256k1 curve"
+            ],
             "type": "string",
             "minLength": 22,
             "maxLength": 22,
             "format": "PEM format"
         },
         "rsa-public-key": {
             "id": "#rsa-public-key",
-            "description": "a 2048-bit RSA public key",
+            "description": [
+                "a 2048-bit RSA public key"
+            ],
             "type": "string",
             "minLength": 22,
             "maxLength": 22,
             "format": "PEM format"
         },
+        "encoded-hash": {
+            "id": "#encoded-hash",
+            "description": [
+                "encoded sha256 hash"
+            ],
+            "type": "string",
+            "minLength": 22,
+            "maxLength": 22,
+            "format": "base64, urlsafe encoded 64 byte array"
```

Comment:
what is minLength and maxLength here? isnt 256 bits ~ length 43 in base64?

Reply:
i believe my comment was specifically that string lengths are not updated and out of scope for this PR.

Reply:
yes, missed that ! thank you